### DISCUSSION TITLE
Card Code GetExecutions - Get By Date Range

### DISCUSSION THREAD

**Discussion by Mikerad1** [Investec-Developer-Community/.github#24] [Ideas]
I don't see this on the documentation, so it would be nice if we can get card code executions by a date range. Not sure how feasible this is and for how long the card code executions last for but yea, would be nice to be able to pull only a subset of the executions instead of all of them. Also not necessarily required to put in a date range

**Comment by programmable-banking-community** [comment]
Thank you for your suggestion! Will pass your feedback on to the teams. They can assess the feasibility of implementing date range filters for card code executions. Will keep you updated.

**Comment by neilwithers** [comment]
This has been added to our backlog. We will get back to you when we have an update.
